libs/numeric/sublibs

```diff
@@ -1 +1 @@
-The existance of this file tells the regression reporting programs that the directory contains sub-directories which are libraries.
\ No newline at end of file
+The existence of this file tells the regression reporting programs that the directory contains sub-directories which are libraries.
```

Comment:
Note that I try not to interfere with missing newlines-at-eof, although I personally side w/ git/github an wanting them to be present.